src/main/resources/mybatis/groundwaterDailyValue.xml

```diff
@@ -71,7 +71,6 @@
           join nwis.monitoring_location
             on gw_daily_value.monitoring_location_id = monitoring_location.monitoring_location_id
          where geom is not null
-           and nwis.monitoring_location.monitoring_location_identifier = #{featureId,jdbcType=VARCHAR}
```

Comment:
This criteria is not needed, since we have already joined based on IDs.  The issue we were troubleshooting on the TEST db involved the IDs and the identifiers not matching, but that is just bad data.

Reply:
Good catch, we are already linking to monitoring_location_identifier in the gw_daily_value
with sub query. Assuming the column (and mon loc id) is correct in groundwater_daily_value,
things are good. In an ideal world, the query could be refactored using the pattern
from the other time series query (with sub query eliminated). The query would
be clearer, more maintainable. Given that the existing query seems good on
speed, probably not worth it.